[2022-11-17T11:20:13.699+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_first_dag_v3.third_task manual__2022-11-17T11:20:10.196694+00:00 [queued]>
[2022-11-17T11:20:13.714+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_first_dag_v3.third_task manual__2022-11-17T11:20:10.196694+00:00 [queued]>
[2022-11-17T11:20:13.714+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-17T11:20:13.714+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-17T11:20:13.714+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-17T11:20:13.736+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): third_task> on 2022-11-17 11:20:10.196694+00:00
[2022-11-17T11:20:13.742+0000] {standard_task_runner.py:55} INFO - Started process 9296 to run task
[2022-11-17T11:20:13.745+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'third_task', 'manual__2022-11-17T11:20:10.196694+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp8pbwz25z']
[2022-11-17T11:20:13.747+0000] {standard_task_runner.py:83} INFO - Job 5: Subtask third_task
[2022-11-17T11:20:13.829+0000] {task_command.py:376} INFO - Running <TaskInstance: my_first_dag_v3.third_task manual__2022-11-17T11:20:10.196694+00:00 [running]> on host 40055b85ab48
[2022-11-17T11:20:13.931+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=my_first_dag_v3
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-17T11:20:10.196694+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-17T11:20:10.196694+00:00
[2022-11-17T11:20:13.932+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-17T11:20:13.933+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo 3rd task running after task 1 and at the same time as task 2']
[2022-11-17T11:20:13.947+0000] {subprocess.py:86} INFO - Output:
[2022-11-17T11:20:13.949+0000] {subprocess.py:93} INFO - 3rd task running after task 1 and at the same time as task 2
[2022-11-17T11:20:13.950+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-17T11:20:13.993+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=third_task, execution_date=20221117T112010, start_date=20221117T112013, end_date=20221117T112013
[2022-11-17T11:20:14.037+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-17T11:20:14.065+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
